_DeploymentAndDistroPackaging/ansible/roles/ciao-launcher/tasks/install.yml

```diff
@@ -37,5 +37,14 @@
       - "{{ agent_key }}"
     notify: restart launcher
 
+  - name: Get OVMF firmware
+    become: no
+    connection: local
+    get_url:
+      url: "{{ firmware_image_url }}"
+      dest: "{{ workdir }}/OVMF.fd"
+      validate_certs: no
+      force: no
+
   - name: Copy OVMF firmware
     copy: src={{ workdir }}/OVMF.fd dest=/usr/share/qemu/OVMF.fd
```

Comment:
Agree with that but for all files not just this one.
I created https://github.com/01org/ciao/issues/1163 to track all files created by the deployment script and ensure they have correct permissions.